pkg/rpc/server_test.go

```diff
@@ -22,9 +21,6 @@ func TestRPC(t *testing.T) {
 
 	chain, handler := initServerWithInMemoryChain(ctx, t)
 
-	defer func() {
-		require.NoError(t, chain.Close())
```

Comment:
curios: probably missed that part: do we expect to close chain(DB) automatically somewhere?

Reply:
Yep, in the `bc.Run()`. After my changes I actually saw double `Close()` invocations that in some situations lead to panicing.